packages/cli/src/cli/index.ts

```diff
@@ -68,6 +68,7 @@ $$$$$$\\   $$$$$$$\\  $$\\  $$$$$$\\   $$$$$$$ |$$\\  $$\\  $$\\  $$$$$$\\  $$$$
       "-t, --template [name]",
       "A template to start your project from. You can use an template repository name from the official thirdweb-example org.",
     )
+    .option("-n, --name [name]", "Name of the new smart contract to create")
```

Comment:
niiiiice - was thinking we should add a step in the flow to name your contract coz its so annoying to have it named `Contract.sol` everytime